src/net.cpp

```diff
@@ -2927,7 +2929,15 @@ std::chrono::seconds CConnman::GetMaxOutboundTimeframe() const
 
 std::chrono::seconds CConnman::GetMaxOutboundTimeLeftInCycle() const
 {
-    LOCK(cs_totalBytesSent);
+    AssertLockNotHeld(m_total_bytes_sent_mutex);
+    LOCK(m_total_bytes_sent_mutex);
+    return _GetMaxOutboundTimeLeftInCycle();
+}
+
+std::chrono::seconds CConnman::_GetMaxOutboundTimeLeftInCycle() const
```

Comment:
nit: I think that the unwritten convention is to put the underscore at the end, like e.g. `AddrManImpl::Good_()`. Or maybe I am mistaken, is there any other place in the code that uses prefix `_`?

Reply:
Changed the underscore to the end. I think only in `AddrManImpl` this is used.